### PR TITLE
fix: IE11 can not close Tabs

### LINKED ISSUE #25153: Tabs 在ie11下无法关闭
- [x] I have searched the [issues](https://github.com/ant-design/ant-design/issues) of this repository and believe that this is not a duplicate.

### Reproduction link
[https://ant.design/components/tabs-cn/#components-tabs-demo-card-top](https://ant.design/components/tabs-cn/#components-tabs-demo-card-top)

### Steps to reproduce
https://ant.design/components/tabs-cn/#components-tabs-demo-card-top

### What is expected?
在ie下 能正常关闭

### What is actually happening?
无法关闭 onRemove不触发 触发了onChange

| Environment | Info |
|---|---|
| antd | 4.3.5 |
| React | 16.13.1 |
| System | window |
| Browser | ie11 |

---
这个问题挺急的 能不能快速发个小版本解决下

### PR DESCRIPTION
**Pull request by zombieJ**
[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md)]

### 🤔 This is a ...

- [ ] New feature
- [x] Bug fix
- [ ] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Perfermance optimization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Other (about what?)

### 🔗 Related issue link

resolve #25153

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |   Fix Tabs can not close in IE 11.        |
| 🇨🇳 Chinese |    修复 Tabs 在 IE 11 下无法关闭的问题。       |

### ☑️ Self Check before Merge

⚠️ Please check all items below before review. ⚠️

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

### CODE REVIEW

File: components/tabs/style/index.less [hunk 1]
```diff
@@ -128,14 +128,29 @@
       margin-left: 0;
     }
 
+    & &-btn,
```
Comment (afc163) [inline components/tabs/style/index.less:131]: &-btn

直接这样可以么。

File: components/tabs/style/index.less [hunk 2]
```diff
@@ -175,6 +171,10 @@
       cursor: not-allowed;
     }
 
+    & &-remove .@{iconfont-css-prefix} {
```
Comment (afc163) [inline components/tabs/style/index.less:174]:
```suggestion
    &-remove .@{iconfont-css-prefix} {
```

Comment (zombieJ) [reply components/tabs/style/index.less:174]: 这个不能删。会被 `tabs .@{iconfont-css-prefix}` 冲掉。

**Comment by zombieJ** [comment]
还是周末发版本~

**Pull request merged**